rtpsender.go

```diff
@@ -199,9 +201,21 @@ func (r *RTPSender) AddEncoding(track TrackLocal) error {
 }
 
 func (r *RTPSender) addEncoding(track TrackLocal) {
+	ssrcFEC, ssrcRTX := SSRC(0), SSRC(0)
```

Comment:
Why not `var ssrcFEC, ssrcRTX SSRC`?

Reply:
pointlessly being clever :) fixed, thanks for the review!